bezier-rs/docs/interactive-docs/src/utils/drawing.ts

```diff
@@ -46,7 +41,17 @@ export const drawText = (ctx: CanvasRenderingContext2D, text: string, x: number,
 	ctx.fillText(text, x, y);
 };
 
-export const drawBezier = (ctx: CanvasRenderingContext2D, points: Point[], dragIndex: number | null = null): void => {
+export const drawBezierHelper = (ctx: CanvasRenderingContext2D, bezier: WasmBezierInstance, stroke = "black", radius = 5): void => {
+	drawBezier(
+		ctx,
+		bezier.get_points().map((p) => JSON.parse(p)),
+		stroke,
+		null,
+		radius
+	);
+};
+
+export const drawBezier = (ctx: CanvasRenderingContext2D, points: Point[], stroke = "black", dragIndex: number | null = null, radius = 5): void => {
```

Comment:
I'd suggest putting `dragIndex` before or after the `stroke` and `radius` parameters because those two both deal with styling and should be contiguous. (If `dragIndex` is the index within `points`, those two parameters should also probably be contiguous.)